src/css/index.scss

```diff
@@ -0,0 +1,418 @@
+$primary_color: #ddd;
+$secondary_color: #fff;
+$btn_border_color: red;
+$btn_hover_color: #ff7675;
+
+* {
+  box-sizing: border-box;
+  font-family: 'montserrat', sans-serif;
+  background: transparent;
+  .slider {
+    display: none;
+  }
+}
+
+.bgimg {
+  background: url('../css/img/cv.jpg');
+  background-size: cover;
+  background-attachment: fixed;
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 100%;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: -1;
+  filter: blur(5px);
+}
+
+.header {
+  max-width: 1000px;
+  width: 100%;
+  padding: 0 auto;
+  height: 100%;
+  margin: auto;
+  a:nth-child(1) {
+    position: relative;
+    left: 0;
+    max-width: 50%;
+    text-decoration: none;
+    font-size: 24px;
+    color: $secondary_color;
+    margin: 0 25px;
+    font-weight: bold;
+  }
+  a:nth-child(2) {
+    display: flex;
+    margin: 0 25px;
+    flex-wrap: wrap;
+    position: relative;
+    max-width: 50%;
+    right: 0;
+    float: right;
+    text-decoration: none;
+    border: 1px solid $btn_border_color;
+    font-size: 18px;
+    padding: 4px 25px;
+    color: $secondary_color;
+  }
+  a:nth-child(2):hover {
+    background: $btn_hover_color;
+  }
+}
+
+.head {
+  width: 100%;
+  position: relative;
+  text-align: center;
+  h1 {
+    font-size: 50px;
+    color: $secondary_color;
+  }
+
+  h2 {
+    text-align: center;
+    color: $primary_color;
+  }
+  img {
+    width: 260px;
+    height: 260px;
+    border-radius: 50%;
+    padding: 10px;
+    background: $secondary_color;
+  }
+}
+
+.slider-section {
+  max-width: 1000px;
+  position: relative;
+  margin: auto;
+  color: #ddd;
+  .slider {
+    .top {
+      width: 100%;
+    }
+    .title {
+      max-width: 50%;
+      position: relative;
+      display: inline-block;
+      margin: 0 20px;
+      color: $secondary_color;
+    }
+    .media {
+      max-width: 50%;
+      position: relative;
+      margin: 0;
+      right: 0;
+      float: right;
+      line-height: 70px;
+      margin: 0 20px;
+      a {
+        text-decoration: none;
+        color: $primary_color;
+        padding: 10px 20px;
+        border: 1px solid $btn_border_color;
+      }
+      a:hover {
+        background: $btn_hover_color;
+      }
+    }
+    .media_unique {
+      max-width: 50%;
+      position: relative;
+      margin: 0;
+      right: 0;
+      float: right;
+      line-height: 70px;
+      margin: 0 30px;
+      a {
+        text-decoration: none;
+        color: $primary_color;
+        padding: 0 8px;
+        font-size: 20px;
+      }
+      a:hover {
+        color: $btn_hover_color;
+      }
+    }
+  }
+}
+
+.bottom {
+  .bottom-content {
+    width: 100%;
+    display: flex;
+    position: relative;
+    .intro {
+      max-width: 50%;
+      width: auto;
+      height: auto;
+      position: relative;
+      display: inline-block;
+      margin: 0 20px 20px;
+    }
+    .experience {
+      max-width: 33.3%;
```

Comment:
Why not assign the `33.3%` to `width`? BTW, I saw you applied `flex` for your bottom-content, so you can set `flex: 1` to each block section for dividing the section width equally.